string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,10 @@
+/* exported isVowel */
+function isVowel(char) {
+  if (char.toLowerCase() === 'a' || char.toLowerCase() === 'e' || char.toLowerCase() === 'i' || char.toLowerCase() === 'o' || char.toLowerCase() === 'u') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+// hi
```

Comment:
This doesn't tell me how to check if a character is a vowel at all 😭 

But hi 👋 😂